### PR TITLE
fix: consistent with the tag component, the processing state uses colorInfo token.

### PR DESCRIPTION
**Pull request by pfdgithub**
tag component status color:
https://github.com/ant-design/ant-design/blob/master/components/tag/style/statusCmp.ts#L36

[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md?plain=1)]

### 🤔 This is a ...

- [ ] New feature
- [x] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Workflow
- [ ] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English | fix: consistent with the tag component, the processing state uses colorInfo token. |
| 🇨🇳 Chinese | fix: 与 tag 组件保持一致，processing 状态使用 colorInfo token 。 |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

File: components/badge/style/index.ts [hunk 1]
```diff
@@ -218,8 +218,8 @@ const genSharedBadgeStyle: GenerateStyle<BadgeToken> = (token) => {
         },
         [`${componentCls}-status-processing`]: {
           overflow: 'visible',
-          color: token.colorPrimary,
-          backgroundColor: token.colorPrimary,
+          color: token.colorInfo,
```
Comment (li-jia-nan) [inline components/badge/style/index.ts:221]: 这两个变量的值是一样的吧？

**Comment by MadCcc** [comment]
feature branch please.

File: components/badge/style/index.ts (hunk 1 above)
Comment (pfdgithub) [reply components/badge/style/index.ts:221]: 默认 token 设置是一样的 ，如果定制了 信息/成功/警告/错误 颜色，就可能不一致了。

**Comment by li-jia-nan** [comment]
多余的 commit 需要清理一下

**Comment by pfdgithub** [comment]
我重新提一个吧

**Pull request closed without merging**